src/Calculator/Views/MainPage.xaml.cs

```diff
@@ -38,6 +38,7 @@ public MainPage()
             m_model.PropertyChanged += OnAppPropertyChanged;
             m_accessibilitySettings = new AccessibilitySettings();
 
+
```

Comment:
Please remove the extra empty lines here and in UnitConverter.xaml.cs

Reply:
done